behaviors/behaviors.py

```diff
@@ -4,6 +4,8 @@
 from django.db import models
 from django.utils import timezone
 
+from slugify import slugify
+
```

Comment:
That seems like a good enough reason to me, we will just place a note in the docs about why we are using it, thanks!